src/main/java/net/dries007/tfc/util/loot/TFCLoot.java

```diff
@@ -37,12 +37,14 @@
 
     public static final LootContextParam<Boolean> ISOLATED = new LootContextParam<>(Helpers.identifier("isolated"));
     public static final LootContextParam<Boolean> PANNED = new LootContextParam<>(Helpers.identifier("panned"));
+    public static final LootContextParam<Boolean> DECAY_HANDLED = new LootContextParam<>(Helpers.identifier("decay_handled"));
 
     public static final RegistryObject<LootItemConditionType> IS_PANNED = lootCondition("is_panned", new PannedCondition.Serializer());
     public static final RegistryObject<LootItemConditionType> IS_ISOLATED = lootCondition("is_isolated", new IsIsolatedCondition.Serializer());
     public static final RegistryObject<LootItemConditionType> ALWAYS_TRUE = lootCondition("always_true", new AlwaysTrueCondition.Serializer());
     public static final RegistryObject<LootNumberProviderType> CROP_YIELD = numberProvider("crop_yield_uniform", new CropYieldProvider.Serializer());
     public static final RegistryObject<LootItemFunctionType> COPY_FLUID = lootFunction("copy_fluid", new CopyFluidFunction.Serializer());
+    public static final RegistryObject<LootItemFunctionType> ROTTING_BLOCK = lootFunction("rotting_block", new RottingBlockLootFunction.Serializer());
```

Comment:
This name doesn't really specify what it does, I'd suggest `copy_decay` and `CopyDecayLootFunction`